tests/integration/targets/aci_vrf_fallback_route_group/tasks/main.yml

```diff
@@ -0,0 +1,411 @@
+    
+    - name: Sort the FBRmembers for the updated fallback group - Check mode
+      ansible.builtin.set_fact:
+        sorted_child_members_cm2: "{{ cm_update_vrf_fbrg.current[0].fvFBRGroup.children | selectattr('fvFBRMember', 'defined') | map(attribute='fvFBRMember.attributes') | list | sort(attribute='rnhAddr') }}"
+
+    - name: Sort children for the updated fallback group - No Check mode
+      ansible.builtin.set_fact:
+        sorted_children2: "{{ nm_update_vrf_fbrg.current[0].fvFBRGroup.children | map('dict2items') | sort(attribute='0.key') | map('items2dict') | list }}"
+    
+    - name: Sort the FBRmembers for the updated fallback group - No Check mode
+      ansible.builtin.set_fact:
+        sorted_child_members2: "{{ nm_update_vrf_fbrg.current[0].fvFBRGroup.children | selectattr('fvFBRMember', 'defined') | map(attribute='fvFBRMember.attributes') | list | sort(attribute='rnhAddr') }}" 
+
+    - name: Verify that vrf_fbrg has been updated correctly in Check modules
+      ansible.builtin.assert:
+        that:
+          - nm_create_vrf_fbrg.current == nm_update_vrf_fbrg.previous
+          - cm_update_vrf_fbrg.proposed.fvFBRGroup.attributes.name == "test_fallback_route_group"
+          - sorted_child_members_cm2.0.rnhAddr == "1.1.1.1"
+          - sorted_child_members_cm2.1.rnhAddr == "2.2.2.2"
+          - sorted_child_members_cm2.2.rnhAddr == "4.4.4.4"
+          - sorted_children_cm2.3.fvFBRoute.attributes.fbrPrefix == "1.1.1.1/2"
+          - nm_update_vrf_fbrg.current.0.fvFBRGroup.attributes.name == "test_fallback_route_group"
+          - sorted_child_members2.0.rnhAddr == "1.1.1.1"
+          - sorted_child_members2.1.rnhAddr == "2.2.2.2"
+          - sorted_child_members2.2.rnhAddr == "4.4.4.4"
+          - sorted_child_members2.3.rnhAddr == "5.5.5.5"
+          - sorted_children2.4.fvFBRoute.attributes.fbrPrefix == "1.1.1.1/2"
+    
+    - name: Update vrf_fallback_route again
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *aci_info
+        tenant: ansible_test
+        vrf: vrf_test
+        name: test_fallback_route_group
+        fallback_route: 1.1.1.1/2
+        fallback_members: 
+          - 5.5.5.5
+          - 2.2.2.2
+          - 1.1.1.1
+          - 4.4.4.4
+        description: Test for Fallback Route Group
+        state: present
+      register: nm_update_vrf_fbrg_again
+      
+    - name: Verify that vrf_fbrg has been updated correctly in Check modules
+      ansible.builtin.assert:
+        that:
+          - nm_update_vrf_fbrg_again is not changed
+          - nm_update_vrf_fbrg_again.current == nm_update_vrf_fbrg.current
+    
+    - name: Delete child from vrf_fallback_route 
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *aci_info
+        tenant: ansible_test
+        vrf: vrf_test
+        name: test_fallback_route_group
+        fallback_route: 2.2.2.2/3
```

Comment:
why test changing the fallback route here while naming it delete from vrf_fallback_route